feature-detects/mathml.js

```diff
@@ -21,8 +21,8 @@ define(['Modernizr', 'testStyles'], function( Modernizr, testStyles ) {
   Modernizr.addTest('mathml', function() {
     var ret;
 
-    testStyles('#modernizr{position:absolute}', function(node){
-      node.innerHTML = '<math><mfrac><mi>xx</mi><mi>yy</mi></mfrac></math>';
+    testStyles('#modernizr{position:absolute;display:inline-block}', function(node){
```

Comment:
I noticed it when validating the caniuse suite in firefox (I just added mathml to the `map`)